pkgs/servers/nitter/default.nix

```diff
@@ -60,5 +61,8 @@ nimPackages.buildNimPackage rec {
     mainProgram = "nitter";
   };
 
-  passthru.tests = { inherit (nixosTests) nitter; };
+  passthru = {
+    tests = { inherit (nixosTests) nitter; };
+    updateScript = unstableGitUpdater {};
+  };
```

Comment:
passthru should be between postInstall and meta

Reply:
Updated.